ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -7,6 +7,8 @@ import Heading from '#app/components/Heading';
 import Text from '#app/components/Text';
 import { ServiceContext } from '#contexts/ServiceContext';
 import nodeLogger from '#lib/logger.node';
+import LegacyText from '#app/legacy/containers/Text';
+import Blocks from '#app/legacy/containers/Blocks';
```

Comment:
Aliases don't meet the component standards. I know there's been some debate around this so lets have the discussion and change the standard or follow it I think. It's important we are consistent even if we don't 100% agree

Reply:
Shall we hold off from merging this then until a decision is made? Just on our rule for Alias's, the NextJS app isn't really covered by our reasoning in the [component standards](https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/coding-standards-react-components--page#typescript-for-frontend-components), due to us using source code from the express app, unless we maintain two versions of code to enable co-location

Reply:
@andrewscfc What do you think?

Reply:
It's a fair point @HarveyPeachey I think we need to revisit this standard at least in the context of the NextJS app. Could you (or maybe @amoore108) raise a PR changing the standard as you see appropriate and we can discuss async? 

Feel free to merge though :)

Reply:
Yep I'll take a look at doing this.